obsidian.el

```diff
@@ -474,6 +702,68 @@ replaced by the link."
         (delete-active-region))
     (insert link-str)))
 
+;;;###autoload
+(defun obsidian-remove-link ()
+  "Remove link and replace with link text."
+  (interactive)
+  (cond ((markdown-link-p)
+         (let ((link (markdown-link-at-pos (point))))
+           (delete-region (nth 0 link) (nth 1 link))
+           (insert (or (nth 2 link) (nth 3 link)))))
+        ((obsidian-wiki-link-p)
+         (markdown-kill-thing-at-point)
+         (yank))))
+
+(defun obsidian-prepare-tags-list (tags)
+  "Prepare a list of TAGS with both lower-case and capitalized versions.
+
+Obsidian Notes tags are case-independent and are therefore considered to be
+the same no matter their case.  Sometimes it's convenient to capitalize a
+tag, for example when using it at the start of the sentence.  This function
+allows completion with both lower and upper case versions of the tags."
+  (let* ((lower-case (-map #'s-downcase tags))
+         (capitalized (-map #'s-capitalize lower-case))
+         (merged (-concat tags lower-case capitalized)))
+    (-distinct merged)))
+
+(defun obsidian-tags-backend (command &rest arg)
+  "Completion backend for company used by obsidian.el.
+Argument COMMAND company command.
+Optional argument ARG word to complete."
+  (interactive (if (and (featurep 'company)
+                        (fboundp 'company-begin-backend))
+                   (company-begin-backend 'obsidian-tags-backend)
+                 (error "Company not installed")))
+  (cl-case command
+    (prefix (and (eq major-mode 'markdown-mode)

```

Comment:
As `obsidian--tag-regex` was updated, the tags backend needed to be updated, as well.  I think this still works as it did previously.